monkCLI/src/commands/checkContainerSizeCommand.php

```diff
@@ -101,7 +155,12 @@ private function checkContainerSize($client, $containerName, OutputInterface $ou
         $progress->finish();
 
         $output->writeln('');
-        $this->message("<info>Total file size is - {$totalSize}.</info>", $output, true);
+        $this->message("<info>
```

Comment:
I would put each of these lines in their own `$this->message()` call.  But you can skip the third parameter so that each line is right under eachother.